recipes/pseudo_dojo/pymatgen.patch

```diff
@@ -0,0 +1,21 @@
+From d35a258f077621d2ff55576476e9d6a8a676d5b8 Mon Sep 17 00:00:00 2001
+From: gmatteo <gmatteo@gmail.com>
+Date: Thu, 17 Oct 2019 19:58:40 +0200
+Subject: [PATCH] drop support for py2, remove six, migrate to flowtk API
```

Comment:
Is this a patch from upstream? This subject line does not seem like what's happening in that patch.

Reply:
The patch included more changes, I focused on the parts required for pymatgen compatibility. I hope they soon release a new version.